tiledb/bioimg/helpers.py

```diff
@@ -159,13 +159,16 @@ def get_axes_translation(
     return {axis: [axis] for axis in axes}
 
 
-def iter_color(attr_type: np.dtype) -> Iterator[Dict[str, int]]:
+def iter_color(attr_type: np.dtype, channel_num: int = 3) -> Iterator[Dict[str, int]]:
     min_val = np.iinfo(attr_type).min
     max_val = np.iinfo(attr_type).max
 
-    yield {"red": max_val, "green": min_val, "blue": min_val, "alpha": max_val}
-    yield {"red": min_val, "green": max_val, "blue": min_val, "alpha": max_val}
-    yield {"red": min_val, "green": min_val, "blue": max_val, "alpha": max_val}
+    if channel_num == 1:
+        yield {"red": max_val, "green": max_val, "blue": max_val, "alpha": max_val}
+    elif channel_num == 3:
+        yield {"red": max_val, "green": min_val, "blue": min_val, "alpha": max_val}
+        yield {"red": min_val, "green": max_val, "blue": min_val, "alpha": max_val}
+        yield {"red": min_val, "green": min_val, "blue": max_val, "alpha": max_val}
```

Comment:
Could you please provide unit tests for `iter_color` it has been modified a lot lately and we want to make sure that we don't patch ad-hoc functions. We need to make sure that the function is complete and functional.